### ISSUE TITLE
Dashboard view entity problems

### ISSUE THREAD

**Issue by magaiverpr** [glpi-project/glpi#7785]
**Describe the bug**

When two (or more) technician is using GLPI, the dashboard bar is show the quantity of the tickets of another entity. If I keep using the GLPI, the bar is updated, but the other technician of another entity goes wrong.

![image](https://user-images.githubusercontent.com/10533823/88808393-351c6700-d189-11ea-9ab8-9100e33cd30d.png)

**To reproduce**

1. Connect with a technician who pertences to an entity.
2. Connect with another technician who works for another entity of the step 1.
3. Look the dashboard bar, click the boxes to update.
4. Look with the other technician, see that the boxes change the number.

**Expected behavior**

Boxes show the wrong quantity of the tickets in the original entity.

**Logs**

```
paste logs here
Find them in *-error.log files under glpi/files/_log/
```

**Screenshots**

**Your GLPI setup (you can find it in Setup > General menu, System tab)**

```
Operating system: Linux ip-wadawada #20-Ubuntu SMP Fri Jul 10 05:52:40 UTC 2020 x86_64
PHP 7.4.3 apache2handler (Core, FFI, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apache2handler, apc, apcu, calendar,
	ctype, curl, date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, imagick, imap, intl, json, ldap, libxml,
	mbstring, memcache, mysqli, mysqlnd, openssl, pcre, pdo_mysql, posix, pspell, readline, session, shmop, sockets, sodium,
	standard, sysvmsg, sysvsem, sysvshm, tidy, tokenizer, xml, xmlreader, xmlrpc, xmlwriter, xsl, zlib)
Setup: max_execution_time="30" memory_limit="128M" post_max_size="8M" safe_mode="" session.save_handler="files"
	upload_max_filesize="2M" 
Software: Apache/2.4.41 (Ubuntu) (Apache/2.4.41 (Ubuntu) Server at wadawada Port 80)
	Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.89 Safari/537.36
Server Software: Ubuntu 20.04
	Server Version: 10.3.22-MariaDB-1ubuntu1
	Server SQL Mode: 
	Parameters: root@127.0.0.1/glpi
	Host info: 127.0.0.1 via TCP/IP
	
PHP version is at least 7.2.0 - Perfect!PHP version is at least 7.2.0 - Perfect!
Sessions support is available - Perfect!Sessions support is available - Perfect!
Allocated memory > 64 Mio - Perfect!Allocated memory > 64 Mio - Perfect!
mysqli extension is installedmysqli extension is installed
ctype extension is installedctype extension is installed
fileinfo extension is installedfileinfo extension is installed
json extension is installedjson extension is installed
mbstring extension is installedmbstring extension is installed
iconv extension is installediconv extension is installed
zlib extension is installedzlib extension is installed
curl extension is installedcurl extension is installed
gd extension is installedgd extension is installed
simplexml extension is installedsimplexml extension is installed
intl extension is installedintl extension is installed
ldap extension is installedldap extension is installed
apcu extension is installedapcu extension is installed
Zend OPcache extension is installedZend OPcache extension is installed
xmlrpc extension is installedxmlrpc extension is installed
CAS extension is installedCAS extension is installed
exif extension is installedexif extension is installed
zip extension is not presentzip extension is not present
bz2 extension is not presentbz2 extension is not present
sodium extension is installedsodium extension is installed
Database version seems correct (10.3.22) - Perfect!Database version seems correct (10.3.22) - Perfect!
Timezones seems loaded in databaseTimezones seems loaded in database
The log file has been created successfully.The log file has been created successfully.
Write access to /var/www/html/glpi/config has been validated.Write access to /var/www/html/glpi/config has been validated.
Write access to /var/www/html/glpi/files has been validated.Write access to /var/www/html/glpi/files has been validated.
Write access to /var/www/html/glpi/files/_dumps has been validated.Write access to /var/www/html/glpi/files/_dumps has been validated.
Write access to /var/www/html/glpi/files/_sessions has been validated.Write access to /var/www/html/glpi/files/_sessions has been validated.
Write access to /var/www/html/glpi/files/_cron has been validated.Write access to /var/www/html/glpi/files/_cron has been validated.
Write access to /var/www/html/glpi/files/_graphs has been validated.Write access to /var/www/html/glpi/files/_graphs has been validated.
Write access to /var/www/html/glpi/files/_lock has been validated.Write access to /var/www/html/glpi/files/_lock has been validated.
Write access to /var/www/html/glpi/files/_plugins has been validated.Write access to /var/www/html/glpi/files/_plugins has been validated.
Write access to /var/www/html/glpi/files/_tmp has been validated.Write access to /var/www/html/glpi/files/_tmp has been validated.
Write access to /var/www/html/glpi/files/_cache has been validated.Write access to /var/www/html/glpi/files/_cache has been validated.
Write access to /var/www/html/glpi/files/_rss has been validated.Write access to /var/www/html/glpi/files/_rss has been validated.
Write access to /var/www/html/glpi/files/_uploads has been validated.Write access to /var/www/html/glpi/files/_uploads has been validated.
Write access to /var/www/html/glpi/files/_pictures has been validated.Write access to /var/www/html/glpi/files/_pictures has been validated.
Write access to /var/www/html/glpi/marketplace has been validated.Write access to /var/www/html/glpi/marketplace has been validated.
Web access to the files directory should not be allowed Check the .htaccess file and the web server configuration.Web access to the files directory should not be allowed
Check the .htaccess file and the web server configuration.
```

**Additional context**

**Comment by magaiverpr** [comment]
Using GLPI 9.5.1

**Comment by orthagh** [comment]
Hello @magaiverpr 

I work on some dashboard improvements (see #7793), and fix the issue in that pr.
The issue is due to we cache a lot for the dashboard to avoid performance issues, and for the current session was not a parameter of the cache entry.

I don't know if the fix will be in the incoming 9.5.2 but i hope to be ready (otherwise it will be in 9.5.3)

**Comment by magaiverpr** [comment]
thanks!
